frontend/src/Editor/QueryPanel/FilterandSortPopup.jsx

```diff
@@ -310,8 +310,8 @@ const MenuButton = ({
   };
 
   return (
-    <div className={`field p-2 ${noMargin ? '' : 'mx-1'} tj-list-btn`}>
-      <Button.UnstyledButton onClick={handleOnClick} disabled={disabled} classNames="d-flex justify-content-between">
+    <div onClick={handleOnClick} className={`field p-2 ${noMargin ? '' : 'mx-1'} tj-list-btn`}>
```

Comment:
instead of adding onclick to div can we remove the extra padding to div and increase the size of button? Adding onClick to div might lead to user accessibility issues like unable to navigate by tab button etc